newstart/index.html

```diff
@@ -34,6 +34,7 @@ <h1 id="main-header" class="screenreader-only">Kids &amp; Teens Get a New Start
           <img src="../img/newstart-header.svg" alt="cartoon graphic displaying Kids &amp; Teens Get a New Start" aria-labelledby="main-header">
         </div>
         <span class="spanish-subhead" lang="es-419">Ni&ntilde;os y adolescentes empieza desde cero en las bibliotecas de N.Y.C.</span>
+        <span class="spanish-subhead" lang="zh">兒童和青少年在紐約市圖書館有一個新的開始</span>
```

Comment:
This might require a new `class` before merging. @ricardoom might need to help with that. Thoughts?